app/src/main/java/com/example/georgesamuel/dubaihotels/activities/DetailActivity.java

```diff
@@ -0,0 +1,111 @@
+package com.example.georgesamuel.dubaihotels.activities;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AlertDialog;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.annotation.SuppressLint;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.example.georgesamuel.dubaihotels.R;
+
+public class DetailActivity extends AppCompatActivity {
+
+    private androidx.appcompat.widget.Toolbar toolbar;
+    private String hotel_image_url;
```

Comment:
fields naming convention doesn't have _ , better to use Capital letters to separate between words 
please see code style 2.2.1 Fields definition and naming but don't start with m or s as google style doesn't recommend that.